### ISSUE TITLE
Passwordless login using a security key doesn't work

### ISSUE THREAD

**Issue by vaio2005** [goauthentik/authentik#7436]
**Describe the bug**
After having setup passwordless login, which I can confirm to be working by using Face ID on my phone, on either iOS or Chrome (Win11), won't allow login with my YubiKey ("This security key doesn't look familiar, please try a different one" error on the Windows security popup)

**To Reproduce**

1. Set up a Webauthn security key on your account
2. Set up a flow as described in the documentation, and apply it to the main login flow
3. Try to log in using the security key added above

**Expected behavior**
The security key login should work normally, as does the Face ID login I have set up.

**Screenshots**
<img width="342" alt="CredentialUIBroker_O653xCOE7F" src="https://github.com/goauthentik/authentik/assets/40716956/63fa6a0e-b265-40f9-b987-3a57601a25f4">
<img width="342" alt="iwmequ8PW2" src="https://github.com/goauthentik/authentik/assets/40716956/8211033c-cbeb-40f3-adc6-71dbd0ae1321">

**Logs**
No relevant logs have been found - Only health checks.

**Version and Deployment (please complete the following information):**

-   authentik version: [2023.10.2](https://goauthentik.io/docs/releases/2023.10)
-   Deployment: Docker compose, under Traefik

**Additional context**
If any additional information/screenshots is required, please let me know.

**Comment by Minionflo** [comment]
no stale

**Comment by jroose** [comment]
This bug is affecting me as well.

**Comment by jroose** [comment]
I think I've figured out a workaround, or maybe this issue is just a documentation problem.  I had to bind an identification stage to my passwordless login flow, so the user has a chance to enter their username.  Then the WebAuthn authentication can proceed as normal.  However, this does lead to a pretty confusing flow of:
1) Being prompted for a username, but clicking the passwordless option instead
3) Being prompted for a username again, entering it here
4) Authenticaiton via WebAuthN
5) Successful login

In short, I think it works, but I think I'll have to do some learning on how to make it work well in Authentik.  Either way, I'm now able to log in using my Yubikey.

**Comment by vaio2005** [comment]
Nostale

**Comment by vaio2005** [comment]
No stale

**Comment by napiat** [comment]
I also had this problem and implemented the workaround by @jroose.

After some more digging, I changed my WebAuthn setup stage settings: "Resident key requirement" and "User verification" was "Preferred", is now "Required". Then I unregistered my WebAuthn device from my account and re-added it.

Now the passwordless flow also works without the additional identification stage.

**Comment by BeryJu** [comment]
We'll be adding some UI improvements and docs improvements to better clarify which settings in the stages should be set to what values for different use-cases/platforms/authenticators

**Comment by ToXinE** [comment]
https://github.com/goauthentik/authentik/issues/7436#issuecomment-2135343626 solve my issue to but now i think I HAVE to set an hardware key for each new users, right ?
